roles/kubernetes/preinstall/tasks/set_facts.yml

```diff
@@ -83,5 +83,17 @@
 - set_fact:
     peer_with_calico_rr: "{{ 'calico-rr' in groups and groups['calico-rr']|length > 0 }}"
 
+- name: check if atomic host
+  stat:
+    path: /run/ostree-booted
+  register: ostree
+
+- set_fact:
+    is_atomic_host: "{{ ostree.stat.exists }}"
+
+- set_fact:
+    kube_cert_group: "kube"
```

Comment:
The user `kube` already comes installed in atomic and is in a [read only file system](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/installation_and_configuration_guide/#managing_user_accounts), this user currently belongs to group `kube`. Adding `kube` user to the group `kube-cert` is failing with ansible user module as it doesn't seem to recognize that user exists in `/usr/lib/passwd` so I am just using the group that kube user already belongs to. 

I definitely would like to understand if there are issues with using `kube` user group.